.github/workflows/preview-env-deploy.yml

```diff
@@ -28,6 +32,18 @@ jobs:
             secret/data/products/camunda-docs/ci/preview-environment PREVIEW_ENV_BUCKET_NAME;
             secret/data/products/camunda-docs/ci/preview-environment PREVIEW_ENV_GCLOUD_SA_KEY;
 
+      - name: Restore docusaurus cache
+        if: github.ref != 'refs/heads/main'
+        uses: actions/cache/restore@v4
+        with:
+          path: |
+            ${{ github.workspace }}/.docusaurus
```

Comment:
Are files in this cache dependent on the base URL? If the base URL on master is going to be different that those for individual PRs, will there still be a speed-up?

Reply:
yeah you are right actually I didnt think of that, let me verify this